table-templates/template-table_02.qmd

```diff
@@ -0,0 +1,113 @@
+---
+title: "FDA Table 2"
+subtitle: "Baseline Demographic and Clinical Characteristics Safety Population, Pooled Analyses"
+format: html
+---
+
+::: panel-tabset
+## Spec. Screenshot
+
+![](images/table_02.png){fig-align="center"}
+
+## Table
+
+```{r, message=FALSE, warning=FALSE, echo=FALSE}
+library(scda)
+library(falcon)
+library(dplyr)
+
+adsl <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "adsl")
+advs <- scda::synthetic_cdisc_dataset("rcd_2022_10_13", "advs")
+
+adsl <- adsl %>%
+  dplyr::mutate(AGEGR1 = as.factor(dplyr::case_when(
+    AGE >= 17 & AGE < 65 ~ ">=17 to <65",
+    AGE >= 65 ~ ">=65",
+    AGE >= 65 & AGE < 75 ~ ">=65 to <75",
+    AGE >= 75 ~ ">=75"
+  )))
+
+advs <- advs %>%
+  dplyr::filter(AVISIT == "BASELINE", VSTESTCD == "TEMP") %>%
+  dplyr::select("USUBJID", "AVAL")
+
+anl <- dplyr::left_join(adsl, advs, by = "USUBJID")
+
+make_table_02(
+  df = anl,
+  vars = c("SEX", "AGE", "AGEGR1", "RACE", "ETHNIC", "COUNTRY", "AVAL"),
+  lbl_vars = c(
+    "Sex", "Age, years", "Age Group, years", "Race", "Ethnicity",
+    "Country of Participation", "Baseline Temperature (C)"
+  )
+)
+```
+
+## Table Setup
+
```

Comment:
I realized that here in the 'Table Setup' tab we have the exact same code as in the 'Table' tab. In case we want to modify something, we need to do it in both places. Would it be possible to "store" the code somewhere else and only call it (in the sense of evaluating) in the 'Table Setup' tab and only echo it in the 'Table Setup' tab?